packages/app/src/cli/models/app/app.test-data.ts

```diff
@@ -1154,7 +1154,7 @@ export function testDeveloperPlatformClient(stubs: Partial<DeveloperPlatformClie
     orgFromId: (_organizationId: string) => Promise.resolve(testOrganization()),
     appsForOrg: (_organizationId: string) => Promise.resolve({apps: [testOrganizationApp()], hasMorePages: false}),
     specifications: (_app: MinimalAppIdentifiers) => Promise.resolve(testRemoteSpecifications),
-    templateSpecifications: (_appId: string) => Promise.resolve(testRemoteExtensionTemplates),
+    templateSpecifications: (_app: MinimalAppIdentifiers) => Promise.resolve(testRemoteExtensionTemplates),
```

Comment:
I'm going to post about this elsewhere, but this is not good -- I saw we did it for specifications too. We are creating a dependency on having a loaded remote app, when _one is not required_. We should look into using a more narrow contract, and ideally one based on client-side loading only. I do realise the app is generally loaded already in the places where this is used today - the point stands, because we are making it harder to create efficiencies in how commands are structured.

Reply:
The org ID won't be present if they haven't connected to an app yet. We don't yet support just fetching an org ID without creating an app. (By the way - does that mean we would store it without storing a client ID? Need to support fetching it next time, etc. Not as simple as it seems on first blush)